src/main/jbake/content/blog/2017/06/23/opentracing-appmetrics.adoc

```diff
@@ -0,0 +1,270 @@
+= Using OpenTracing to collect Application Metrics in Kubernetes
+Gary Brown
+2017-6-23
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, kubernetes, prometheus
+
+This article will show how http://opentracing.io/[OpenTracing] instrumentation can be used to
```

Comment:
As @pilhuhn would say: it would probably be a good idea to add a teaser here :) . 

It could perhaps also be useful to add a bit more of information on what the end goal might be, as "collecting application metrics" is not a goal per se. Something like: "... collect Application Metrics... , so that this can be added to your internal dashboard or connected to an alerting system."